homeassistant/components/netatmo/__init__.py

```diff
@@ -105,6 +106,25 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     await data_handler.async_setup()
     hass.data[DOMAIN][entry.entry_id][DATA_HANDLER] = data_handler
 
+    async def handle_event(event):
+        """Handle webhook events."""
+        if event["data"]["push_type"] == "webhook_activation":
```

Comment:
How often can this happen? Should you unsubscribe this listener after forwarding it once?

Reply:
If the webhook is re-registered because for example the webhook was banned by Netatmo.

Reply:
Since you don't unsubscribe from the dispatcher, can you end up calling this twice?

Reply:
This is now solved by unsubscribing from the dispatcher.